src/map/pc.cpp

```diff
@@ -9235,6 +9235,18 @@ int pc_setregistry(struct map_session_data *sd, int64 reg, int val)
 	return 1;
 }
 
+//maybe move this as intif or script ?
+bool pc_check_RegistryVariableLenght(struct map_session_data *sd, int pType, const char *val, size_t* vlen) 
```

Comment:
Typo in "length"

Reply:
which typo. I wrote it lije I wrote lighta =) jk, will fix.

Reply:
`lighta != lengtha`